src/reflog.rs

```
@@ -0,0 +1,379 @@
+}
+
+#[napi]
+impl ReflogEntry {
+  #[napi]
+  /// Get the committer of this reflog entry.
+  ///
+  /// @category ReflogEntry/Methods
+  ///
+  /// @signature
+  /// ```ts
+  /// class ReflogEntry {
+  ///   committer(): Signature;
+  /// }
+  /// ```
+  ///
+  /// @returns Committer signature of this reflog entry.
+  pub fn committer(&self) -> crate::Result<Signature> {
+    Signature::try_from(self.inner.committer())
+  }
+
+  #[napi]
+  /// Get the new object ID (SHA1) of this reflog entry.
+  ///
+  /// @category ReflogEntry/Methods
+  ///
+  /// @signature
+  /// ```ts
+  /// class ReflogEntry {
+  ///   idNew(): string;
+  /// }
+  /// ```
+  ///
+  /// @returns New object ID (SHA1) of this reflog entry.
+  pub fn id_new(&self) -> String {
+    self.inner.id_new().to_string()
+  }
+
+  #[napi]
+  /// Get the old object ID (SHA1) of this reflog entry.
+  ///
+  /// @category ReflogEntry/Methods
+  ///
+  /// @signature
+  /// ```ts
+  /// class ReflogEntry {
+  ///   idOld(): string;
+  /// }
+  /// ```
+  ///
+  /// @returns Old object ID (SHA1) of this reflog entry.
+  pub fn id_old(&self) -> String {
+    self.inner.id_old().to_string()
+  }
+
+  #[napi]
+  /// Get the message of this reflog entry.
+  ///
+  /// @category ReflogEntry/Methods
+  ///
+  /// @signature
+  /// ```ts
+  /// class ReflogEntry {
+  ///   message(): string | null;
+  /// }
+  /// ```
+  ///
+  /// @returns Message of this reflog entry. Returns `null` if no message is present.
+  pub fn message(&self) -> Option<String> {
+    self.inner.message().map(|x| x.to_string())
+  }
+
+  #[napi]
+  /// Get the message bytes of this reflog entry.
+  ///
+  /// @category ReflogEntry/Methods
+  ///
+  /// @signature
+  /// ```ts
+  /// class ReflogEntry {
+  ///   messageBytes(): Uint8Array | null;
+  /// }
+  /// ```
+  ///
+  /// @returns Message bytes of this reflog entry. Returns `null` if no message is present.
+  pub fn message_bytes(&self) -> Option<Uint8Array> {
```

I think this method seems unnecessary, as it's rare case to handle messages as a uint8array.

Instead of returning a nullable message in `message()` method, how about throw an error if the message isn't in UTF-8?

e.g.
```rs
pub fn message(&self) -> crate::Result<Option<String>> {
  // ... throw if message is not utf-8
}
```